TypeCobol/Compiler/CodeElements/Entry/DataType.cs

```diff
@@ -173,6 +173,7 @@ public static Nodes.TypeDefinition CreateBuiltIn(DataType type)
 		}
 		private static Nodes.TypeDefinition CreateBase(DataType type) {
 			var entry = new DataTypeDescriptionEntry();
+		    entry.Visibility = AccessModifier.Public;
```

Comment:
Both Visibilities can be set in the method CreateBuiltIn

Reply:
See 924ed27d5967cdb01052e052ad645d8c7f7dec8a